src/IB/IMPMethod.cpp

```diff
@@ -127,7 +126,7 @@ kernel(const double X,
        boost::multi_array<double, 1>& dphi)
 {
     const double X_o_dx = (X - patch_x_lower) / dx;
-    stencil_box_lower = boost::math::round(X_o_dx) + patch_box_lower - kernel_width;
+    stencil_box_lower = int(std::round(X_o_dx)) + patch_box_lower - kernel_width;
```

Comment:
`static_cast<int>`? And, if it needs a cast anyway, should we just omit the call to `std::round` and let the cast do the rounding for us?